.pre-commit-config.yaml

```diff
@@ -95,10 +95,10 @@ repos:
         entry: pyupgrade --py39-plus --keep-runtime-typing
         types:
           - python
-  - repo: https://github.com/gitleaks/gitleaks
-    rev: v8.18.4
-    hooks:
-      - id: gitleaks
+  # - repo: https://github.com/gitleaks/gitleaks
+  #   rev: v8.18.4
+  #   hooks:
+  #     - id: gitleaks
```

Comment:
@kirtimanmishrazipstack why is this commented?

Reply:
@ritwik-g github workflow is failing due to this. I think I need to install go like python inside to make it work